models/src/main/scala/coop/rchain/models/manual/DeployId.scala

```diff
@@ -0,0 +1,5 @@
+package coop.rchain.models.manual
+
+final case class DeployId(
+                           sig: _root_.com.google.protobuf.ByteString = _root_.com.google.protobuf.ByteString.EMPTY
```

Comment:
here as well, please use imports to have just `sig: ByteString = ByteString.EMPTY`

Reply:
IntelliJ can help working/inserting imports automatically.
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html